docs/mkdocs/docs/technical/contributing.md

```diff
@@ -1,19 +1,88 @@
-You can download the output [here](https://github.com/man-group/ArcticDB/pkgs/container/cibuildwheel_manylinux).
+```bash
+docker pull ghcr.io/man-group/cibuildwheel_manylinux:2.12.1-3a897
+docker run -it ghcr.io/man-group/cibuildwheel_manylinux:2.12.1-3a897
+```
+
+> :warning: The below instructions do not have to be run in the provided docker image. They can be run against any Python installation on any Linux distribution as long as the basic build [dependencies are available](#setting-up-linux).
+> 
+> If running outside the provided docker image, please change `/opt/python/cp39-cp39/bin/python3` in the examples below 
+> to an appropriate path for Python.
+
+#### 2) Check out ArcticDB including submodules
+
+```bash
+cd 
+git clone https://github.com/man-group/ArcticDB.git
+cd ArcticDB
+git submodule init && git submodule update
+```
+
+#### 3) Kick off the build
+
+```bash
+/opt/python/cp39-cp39/bin/python3 -m pip install -U pip setuptools wheel grpcio-tools
+ARCTIC_CMAKE_PRESET=skip /opt/python/cp39-cp39/bin/python3 setup.py develop
+# Change the below Python_EXECUTABLE value to build against a different Python version
+cmake -DPython_EXECUTABLE=/opt/python/cp39-cp39/bin/python -DTEST=off --preset linux-debug cpp
+# NOTE: Set parallel level below to number of cores!
+cmake --build --parallel 8 --preset linux-debug cpp
+```
+
+#### 4) Run ArcticDB
+
+Ensure the below is run in the Git project root:
+
+```bash
+# PYTHONPATH first part = Python module, second part compiled C++ binary
+PYTHONPATH=`pwd`/python:`pwd`/cpp/out/linux-debug-build/arcticdb/ /opt/python/cp39-cp39/bin/python3
+```
+
+Inside the Python shell:
+
+```python
+from arcticdb import Arctic
+```
+
+FAQ
+===
+
+#### How do I build against different Python versions?
+
+Run `cmake` (configure, not build) with either:
+
+1. A different version of Python as the _first_ version of Python on your PATH or...
+2. Point the `Python_EXECUTABLE` CMake variable to a different Python binary
+
+Note that to build the ArcticDB C++ tests **you must have the Python static library available in your installation!**
+
+#### How do I run the Python tests?
 
-**We recommend you use this image for compilation and testing.**
+See [running tests below](#running-python-tests).
 
-Linux
------
+#### How do I run the C++ tests?
+
+See [running tests below](#running-c-tests).
+
+Detailed Build Information
+==========================
+
+Docker Image Construction
+-------------------------
+
+The above docker image is built form [ManyLinux](https://github.com/pypa/manylinux). 
+Build script located [here](https://github.com/man-group/ArcticDB/blob/master/build_tooling/build_many_linux_image.sh).
+
+GitHub output [here](https://github.com/man-group/ArcticDB/pkgs/container/cibuildwheel_manylinux).
+
+**We recommend you use this image for compilation and testing!**
+
+Setting up Linux
+----------------
 The codebase and build system can work with any reasonably recent Linux distribution with at least
```

Comment:
Not your bit of the docs but maybe we should be more proscriptive:
"We support GCC 10, CMake 3.21+ and MSVC v142+"